docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    energy_axis_true = MapAxis.from_energy_bounds(
</pre><p>you might increase the binning in true energy to ensure sampling is good enough.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Done, I also have re-generated the DL3 files with more true energy bins (same PR adding `obs` and `hdu` files).